batch/extension-jberet/src/main/java/org/wildfly/extension/batch/jberet/BatchSubsystemDefinition.java

```diff
@@ -84,7 +84,7 @@
     static final SimpleAttributeDefinition DEFAULT_THREAD_POOL = SimpleAttributeDefinitionBuilder.create("default-thread-pool", ModelType.STRING, true)
             .setAllowExpression(false)
             .setAttributeGroup("environment")
-            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE)
+            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE) //todo this looks like it is missing capability reference
```

Comment:
How can a capability be added for this? Unless something changed with the thread pool utilities in core you can't register them as capabilities.